File: CS/1431_Kids_With_the_Greatest_Number_of_Candies.cs
public class Solution {
    public IList<bool> KidsWithCandies(int[] candies, int extraCandies) {
        int max = Int32.MinValue;
        for(int i = 0; i < candies.Length; i++){
            max = Math.Max(max, candies[i]);
        }
        IList<bool> result = new List<bool>();
        for(int i = 0; i < candies.Length; i++){
            if(candies[i] + extraCandies >= max){
                result.Add(true);
            }
            else{
                result.Add(false);
            }
        }
        return result;
    }
}

File: Python3/1431_Kids_With_the_Greatest_Number_of_Candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result = []
        m = max(candies)
        for i in range(0, len(candies)):
            if candies[i] + extraCandies >= m:
                result.append(True);
            else:
                result.append(False);
        return result
